### PR TITLE
ft view at top of stream page

### PR DESCRIPTION
**Pull request by andygnewman**
Picking the first 'opinion / analysis' piece from the first page of the stream and putting it to the top of the page on the top level stream pages (except Opinion and Life & Arts.

screen shot 2015-07-08 at 12 18 31

### CODE REVIEW

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-08/11:27-2176/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-08/11:27-2176/successes/index.html)

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-08/14:58-2190/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-08/14:58-2190/successes/index.html)

**Comment by andygnewman** [comment]
@i-like-robots can you give this the thumbs up if you're happy

File: server/lib/ft-view-top.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+function FtViewTop(streamItems) {
```
Comment (i-like-robots) [inline server/lib/ft-view-top.js]: As there is only one method I don't think the constructor function is necessary, just export and use the `extractFirstFtView` method

File: views/partials/ft-view-top.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<section class="ft-view" role="section">
+	<h2 class="stream__title">Latest FT comment</h2>
+  <li class="stream-item stream-item--row"  data-video-ids="{{videoIds}}">
```
Comment (i-like-robots) [inline views/partials/ft-view-top.html]: mixed whitespace

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-09/11:25-2194/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-09/11:25-2194/successes/index.html)

File: server/lib/ft-view-top.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+// function FtViewTop(streamItems) {
+// 	this.streamItems = streamItems;
+// }
```
Comment (i-like-robots) [inline server/lib/ft-view-top.js]: May as well remove these commented lines

File: server/lib/ft-view-top.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+		}
+		var metaData = element.raw.item.metadata;
+		var genreName = metaData.genre[0] && metaData.genre[0].term.name;
+		var topicName = metaData.brand[0] && metaData.brand[0].term.name;
+
+		if (genreName === 'Comment' && excludedTopics.indexOf(topicName) === -1) {
+			return element;
+		} else {
+			return false;
+		}
```
Comment (i-like-robots) [inline server/lib/ft-view-top.js]: This `else` block is redundant, it should be fine without it.

**Comment by i-like-robots** [comment]
Two minor comments, up to you if you want to amend :+1:

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-09/11:44-2196/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-09/11:44-2196/successes/index.html)

**Pull request merged**